Makefile

```diff
@@ -39,6 +49,37 @@ fmt-check:
 # For convenience. If this passes then it should pass in GitHub
 ci: fmt-check quiet-test
 
+#--------------------------------------------------------------------
+
+clean-data:
+	@rm -rf $(DATA_DIR)
+
+# Avoid a "feels like a bad day.." violation
+dummy-config:
+	@mkdir -p $(DATA_DIR)/config/policy
+	@echo '{"non_blocking_checks":["not_useful"]}' | jq > $(DATA_DIR)/config/policy/data.json
+
+# Set IMAGE as required like this:
+#  make fetch-attestation IMAGE=...
+#
+# The format and file path is intended to match what is used in the
+# verify-attestation-with-policy script in the build-definitions repo
+# so you can test your rules as they would be applied by the
+# verify-enterprise-contract task.
+#
+
+ifndef IMAGE
+  IMAGE="quay.io/lucarval/tekton-test:latest"
```

Comment:
Should this be here? https://quay.io/organization/hacbs-contract

Reply:
We could use anything with at attestation, but I think this is what Luiz was using for testing the task, so I figured it's as good as any. Though it might be wise to specify a sha in case a future push doesn't come with an attestation.

Reply:
I'm expecting user would specify their own as required. The default is just for convenience/laziness.

Reply:
Should the public key be added to this repo?

Reply:
OK, makes sense

Reply:
I added some comments and switched to a sha.

Reply:
I tagged the image with an additional tag so it's not garbage collected when the `latest` tag moves to a different image.

Reply:
Oh, I didn't think of that. Thanks.

Reply:
Now I think maybe just latest was better, but never mind. If we think of a better default later we'll change it.